Stratis.Bitcoin/Features/RPC/Controllers/ConnectionManagerController.cs

```diff
@@ -35,5 +39,60 @@ public bool AddNode(string endpointStr, string command)
             }
             return true;
         }
+
+        /// <summary>
+        /// RPC implementation of getpeerinfo
+        /// </summary>
+        /// <returns>List of RPC peer nodes</returns>
+        [ActionName("getpeerinfo")]
+        public List<PeerNodeModel> GetPeerInfo()
```

Comment:
Could you please add a list to the cpp code on github?
This way it'll be easier for people to see the source.

https://github.com/bitcoin/bitcoin/blob/0.14/src/rpc/net.cpp